package.json

```diff
@@ -13,7 +13,7 @@
   },
   "dependencies": {
     "@babel/compat-data": "^7.11.0",
-    "@bcrs-shared-components/certify": "1.0.3",
+    "@bcrs-shared-components/certify": "1.0.7",
```

Comment:
Did you verify that this newer version of the Certify component works with the existing code (ie, a Correction)?

If it all works fine then you will have also resolved /bcgov/entity#6208... in that case please move 6208 to Ready for QA when ready ;)

Reply:
Yes it does, and yes will do